draft-ietf-core-oscore-edhoc.md

```diff
@@ -577,7 +580,7 @@ Initial entries in this registry are as listed in {{pre-reg}}.
 
 ## Expert Review Instructions ## {#review}
 
-The IANA registry established in this document is defined as "Expert Review". This section gives some general guidelines for what the experts should be looking for; but they are being designated as experts for a reason, so they should be given substantial latitude.
+The IANA registry established in this document is defined as "Standards Action with Expert Review" or "Specification Required". This section gives some general guidelines for what the experts should be looking for; but they are being designated as experts for a reason, so they should be given substantial latitude.
```

Comment:
Is there a condition on that "or"?  
(I.e., who gets to choose?)

Reply:
Fixed in https://github.com/core-wg/oscore-edhoc/commit/9f1cbd4e511ebfe91b9105fac08609d724b92a35